spec/structs.md

```diff
@@ -2,376 +2,440 @@
 
 ## General
 
-Structs are similar to classes in that they represent data structures that can contain data members and function members. However, unlike classes, structs are value types and do not require heap allocation. A variable of a struct type directly contains the data of the struct, whereas a variable of a class type contains a reference to the data, the latter known as an object.
+Structs are similar to classes in that they represent data structures that can contain data members and function members. However, unlike classes, structs are value types and do not require heap allocation. A variable of a `struct` type directly contains the data of the `struct`, whereas a variable of a `class` type contains a reference to the data, the latter known as an object.
 
-\[*Note*: Structs are particularly useful for small data structures that have value semantics. Complex numbers, points in a coordinate system, or key-value pairs in a dictionary are all good examples of structs. Key to these data structures is that they have few data members, that they do not require use of inheritance or referential identity, and that they can be conveniently implemented using value semantics where assignment copies the value instead of the reference. *end note*\]
+> [!NOTE]
+> [*Note*: Structs are particularly useful for small data structures that have value semantics. Complex numbers, points in a coordinate system, or key-value pairs in a dictionary are all good examples of structs. Key to these data structures is that they have few data members, that they do not require use of inheritance or referential identity, and that they can be conveniently implemented using value semantics where assignment copies the value instead of the reference. *end note*]
 
-As described in §9.3.5, the simple types provided by C\#, such as int, double, and bool, are, in fact, all struct types.
+As described in §9.3.5, the simple types provided by C#, such as `int`, `double`, and `bool`, are, in fact, all `struct` types.
 
 ## Struct declarations
 
 ### General
 
-A *struct-declaration* is a *type-declaration* (§14.7) that declares a new struct:
+A *struct-declaration* is a *type-declaration* (§14.7) that declares a new `struct`:
 
-[]{#Grammar_struct_declaration .anchor}struct-declaration:\
-attributes~opt~ struct-modifiers~opt~ *partial*~opt~ *struct* identifier type-parameter-list~opt\
-~struct-interfaces~opt~ type-parameter-constraints-clauses~opt~ struct-body *;*~opt~
+[#Grammar_struct_declaration](struct-declaration)
+```ANTLR
+struct-declaration:
```

Comment:
Not your edit, but somehow this grammar production really got mangled in the markdown production. Lines 20 - 22 should be replaced with the following: 
```suggestion
struct_declaration
    : attributes? struct_modifier* 'partial'? 'struct' identifier type_parameter_list?
      struct_interfaces? type_parameter_constraints_clause* struct_body ';'?
    ;
```

Reply:
@BillWagner this ANTLR code has been updated.